source/includes/_platform-payments.md

```diff
@@ -158,198 +153,179 @@ Updates a specific set of stored account details. This will trigger an AML check
-| officerList.items.address.addressLine2      | string | Address Line 2                                 | required |
-| officerList.items.address.careOf            | string | Care Of                                        | required |
-| officerList.items.address.country           | string | Country                                        | required |
-| officerList.items.address.locality          | string | Locality                                       | required |
-| officerList.items.address.poBox             | string | Po Box                                         | required |
-| officerList.items.address.postalCode        | string | Postal Code                                    | required |
-| officerList.items.address.region            | string | Region                                         | required |
-
-## `POST /platformApi/wallet`
-```http
+| Name              | Type   | Description                                        | Required |
+| ----------------- | ------ | -------------------------------------------------- | -------- |
+| ukCompany.number  | string | The Company Number registered with Companies House.| required |
+| ukCompany.name    | string | Company Name                                       | required |
 
-POST /platformApi/wallet HTTP/1.1
+### Response
+| Name              | Type   | Description                                        |
+| ----------------- | ------ | -------------------------------------------------- |
+| partyId           | string                                                      |
+| ukCompany.number  | string | The Company Number registered with Companies House.|
+| ukCompany.name    | string | Company Name                                       |
+
+## `GET /companies 🚧`
+```http
+GET /companies HTTP/1.1
 Host: api-sandbox.goji.investments
 Content-Type: application/json
 Authorization: Basic ...
+```
 
-{
-  "walletName" : "Housing Project 22"
-}
+```http 
+HTTP/1.1 200 OK
+Content-Type: application/json
 
+[
+  {
+    "partyId": "COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502"
+    "ukCompany": {
+      "number": "00000000",
+      "name": "Fake Company LIMITED"
+    }
+  },
+  {
+    "partyId": "COM~0176cb07-30ba-4802-ab4a-b0835222cf5b"
+    "ukCompany": {
+      "number": "00000001",
+      "name": "Other Fake Company LIMITED"
+    }
+  }
+]
+```
+### Description
+
+Retrieves all companies that have been registered.
+
+### Response
+| Name              | Type   | Description                                        |
+| ----------------- | ------ | -------------------------------------------------- |
+| partyId           | string                                                      |
+| ukCompany.number  | string | The Company Number registered with Companies House.|
+| ukCompany.name    | string | Company Name                                       |
+
+
+## `GET /companies/{id} 🚧`
+```http
+GET /companies/COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502 HTTP/1.1
```

Comment:
is the `~` just an example? Wondering how its going to behave being in a URL

Reply:
It’s a concrete example for how we were proposing we’d return a PartyId. I checked though, ._-~ are all URL safe.

Reason we want to do this, is that we might vend multiple different types of PartyId, and thought it would be useful to namespace them to look up details related to them. Considered having an idSrc but thought it might be easier for platforms to just receive an ID they can plug into calls, but gives us a bit more information on how to treat them